tests/component/system/test_system.py

```diff
@@ -0,0 +1,12 @@
+from nidaqmx.constants import PowerUpChannelType
+
+
+def test___get_analog_power_up_states___returns_power_up_states(system):
```

Comment:
Add `_with_output_type` to distinguish this from tests for `get_analog_power_up_states` (if we write any).